corehq/apps/hqwebapp/signals.py

```diff
@@ -0,0 +1,32 @@
+from datetime import date
+
+from django.contrib.auth.signals import user_logged_in, user_login_failed
+from django.dispatch import receiver
+
+from corehq.apps.users.models import CouchUser
+
+
+def clear_login_attempts(user):
+    user = CouchUser.from_django_user(user)
+    if user and user.is_web_user() and user.login_attempts > 0:
```

Comment:
This lockout condition block is being used in multiple places, if it is replaced by a simple function or method making changes to this condition latter can be done in just one place.